### PR TITLE
[docs] [migrate] added more known issues 

### PR DESCRIPTION
**Pull request by aishwarya24**
Summary 
- This PR addresses a bunch of known issues: [689](https://github.com/orgs/yugabyte/projects/10/views/11?pane=item&itemId=17435114), [705](https://github.com/orgs/yugabyte/projects/10/views/11?pane=item&itemId=17437161), [707](https://github.com/orgs/yugabyte/projects/10/views/11?pane=item&itemId=17440521), [708](https://github.com/orgs/yugabyte/projects/10/views/11?pane=item&itemId=17440919), [698](https://github.com/orgs/yugabyte/projects/10/views/11?pane=item&itemId=17492026), [699](https://github.com/orgs/yugabyte/projects/10/views/11?pane=item&itemId=17492179), [690](https://github.com/yugabyte/yb-voyager/issues/690), and [621](https://github.com/orgs/yugabyte/projects/10/views/11?pane=issue&itemId=17566508).

- Addressed issue related to `source-db-password` and `target-db-password`- https://github.com/yugabyte/yb-voyager/issues/377.
 I have included a comment beside every code block that uses these arguments and also updated the cli page .

- Removed issue [573](https://github.com/yugabyte/yb-voyager/issues/573) from the list of MySQL issues.

- Deploy preview links for the known issues: [689](https://deploy-preview-15527--infallible-bardeen-164bc9.netlify.app/preview/migrate/known-issues/#incorrect-parsing-of-views-involving-functions-without-alias),  [690](https://deploy-preview-15527--infallible-bardeen-164bc9.netlify.app/preview/migrate/known-issues/#datatype-mismatch-in-objects-causing-issues), [698](https://deploy-preview-15527--infallible-bardeen-164bc9.netlify.app/preview/migrate/known-issues/#tables-partitioned-with-expressions-cannot-contain-primary-unique-keys), [699](https://deploy-preview-15527--infallible-bardeen-164bc9.netlify.app/preview/migrate/known-issues/#multi-column-partition-by-list-is-not-supported), [705](https://deploy-preview-15527--infallible-bardeen-164bc9.netlify.app/preview/migrate/known-issues/#drop-temporary-table-statements-are-not-supported), [707](https://deploy-preview-15527--infallible-bardeen-164bc9.netlify.app/preview/migrate/known-issues/#key-defined-for-a-table-in-functions-procedures-cause-issues), [708](https://deploy-preview-15527--infallible-bardeen-164bc9.netlify.app/preview/migrate/known-issues/#multiple-declarations-of-variables-in-functions).
 
- Renamed Troubleshoot tile on the Voyager landing page to known issues.

### CODE REVIEW

File: docs/content/preview/migrate/known-issues.md [hunk 1]
```diff
@@ -253,6 +208,357 @@ CREATE TABLE address (
...
+```
+
+Choose one from the following suggested changes to the schema.
+
+- Remove the key from the schema as follows:
+
+    ```sql
+    CREATE OR REPLACE FUNCTION foo (p_id integer) RETURNS varchar AS $body$
+      BEGIN
+        CREATE TEMPORARY TABLE temp(id int, name text,key(id));
```
Comment (shubham-yb) [inline docs/content/preview/migrate/known-issues.md:423]: Please remove the `key` from this statement.

Comment (aishwarya24) [reply docs/content/preview/migrate/known-issues.md:423]: good catch. Thanks!

File: docs/content/preview/migrate/migrate-steps.md [hunk 2]
```diff
@@ -147,7 +147,7 @@ yb-voyager export schema --export-dir <EXPORT_DIR> \
         --source-db-type <SOURCE_DB_TYPE> \
         --source-db-host <SOURCE_DB_HOST> \
         --source-db-user <SOURCE_DB_USER> \
-        --source-db-password <SOURCE_DB_PASSWORD> \
+        --source-db-password <SOURCE_DB_PASSWORD> \ # Specify the password in single quotes, if it contains special characters.
```
Comment (ddhodge) [inline docs/content/preview/migrate/migrate-steps.md:150]:
```suggestion
        --source-db-password <SOURCE_DB_PASSWORD> \ # Enclose the password in single quotes if it contains special characters.
```

File: docs/content/preview/migrate/migrate-steps.md [hunk 3]
```diff
@@ -196,7 +196,7 @@ yb-voyager export data --export-dir <EXPORT_DIR> \
...
```
Comment (ddhodge) [inline docs/content/preview/migrate/migrate-steps.md:199]: and for the rest

**Review by ddhodge — APPROVED** [review]

File: docs/content/preview/migrate/known-issues.md [hunk 4]
```diff
@@ -536,13 +952,100 @@ PRIMARY KEY (employee_id, hire_date)) PARTITION BY RANGE (hire_date) ;
...
+
+**GitHub links**: [Issue #636](https://github.com/yugabyte/yb-voyager/issues/636), [Issue #688](https://github.com/yugabyte/yb-voyager/issues/688), [Issue #702](https://github.com/yugabyte/yb-voyager/issues/702)
+
+**Description**: If you define complex names for your source database tables/functions/procedures using backticks or double quotes for example, \`abc xyz\` , \`abc@xyz\`, or "abc@123", the migration hangs during the export data step.
+
+**Workaround**: Rename the objects (Tables/functions/procedures) on the source database to something without special characters.
+
+**Example**
+
+An example schema on the source PostgreSQL database is as follows:
```
Comment (shubham-yb) [inline docs/content/preview/migrate/known-issues.md:976]: This is source MySQL database. Please change it

File: docs/content/preview/migrate/known-issues.md [hunk 5]
```diff
@@ -536,13 +952,100 @@ PRIMARY KEY (employee_id, hire_date)) PARTITION BY RANGE (hire_date) ;
...
+INSERT INTO `xyz abc` VALUES(3);
+```
+
+The exported schema is as follows:
+
+```sql
+CREATE TABLE "xyz abc" (id bigint);
+```
+
+The preceding example will result in an error as follows:
```
Comment (shubham-yb) [inline docs/content/preview/migrate/known-issues.md:991]: These are our backend logs. The user doesn't need to go into such details. We may just mention something generic like it'll throw an error or hang.

File: docs/content/preview/migrate/known-issues.md [hunk 6]
```diff
@@ -536,13 +952,100 @@ PRIMARY KEY (employee_id, hire_date)) PARTITION BY RANGE (hire_date) ;
...
+2022-11-30 05:16:29 INFO ora2pg_export_data.go:76 Executing command: ora2pg -q -t COPY -P 4 -o data.sql -b /home/centos/export-dir/data -c /home/centos/export-dir/temp/.ora2pg.conf --no_header
+2022-11-30 05:16:29 INFO common.go:79 After updating data file paths, TablesProgressMetadata:&{TableSchema:automation TableName:xyz abc FullTableName:automation.xyz abc InProgressFilePath:/home/centos/export-dir/data/tmp_xyz abc_data.sql FinalFilePath:/home/centos/export-dir/data/xyz abc_data.sql Status:0 CountLiveRows:0 CountTotalRows:3 FileOffsetToContinue:0 IsPartition:false ParentTable:}
+
+2022-11-30 05:16:30 INFO ora2pg_export_data.go:92 ora2pg STDOUT: ""
+2022-11-30 05:16:30 ERROR ora2pg_export_data.go:93 ora2pg STDERR: ""
+```
+
+---
+
+#### Import issue with case-senstivie schema names
```
Comment (shubham-yb) [inline docs/content/preview/migrate/known-issues.md:1007]: Spelling error

File: docs/content/preview/migrate/known-issues.md [hunk 7]
```diff
@@ -536,13 +952,100 @@ PRIMARY KEY (employee_id, hire_date)) PARTITION BY RANGE (hire_date) ;
...
 
 ---
+
+#### Exporting data with names for tables/functions/procedures using special characters fails
+
+**GitHub links**: [Issue #636](https://github.com/yugabyte/yb-voyager/issues/636), [Issue #688](https://github.com/yugabyte/yb-voyager/issues/688), [Issue #702](https://github.com/yugabyte/yb-voyager/issues/702)
+
+**Description**: If you define complex names for your source database tables/functions/procedures using backticks or double quotes for example, \`abc xyz\` , \`abc@xyz\`, or "abc@123", the migration hangs during the export data step.
+
+**Workaround**: Rename the objects (Tables/functions/procedures) on the source database to something without special characters.
```
Comment (shubham-yb) [inline docs/content/preview/migrate/known-issues.md:972]: Can we mention `special characters/whitespaces` as whitespaces are a more common use case. Thanks.

**Review by shubham-yb — APPROVED** [review]

**Pull request merged**